pkgs/applications/graphics/paraview/default.nix

```
@@ -35,7 +35,7 @@ in stdenv.mkDerivation rec {
     owner = "paraview";
     repo = "paraview";
     rev = "v${version}";
-    sha256 = "sha256-WvkKGl5lG+apX6m4ULVZZVtDsSUjEVXe/seh95b+LmI=";
+    sha256 = "sha256-LatNHfiAqB2kqzERRnYae0WIXBb4nXQ79Be4kuh8NFQ=";
```

```suggestion
    hash = "sha256-LatNHfiAqB2kqzERRnYae0WIXBb4nXQ79Be4kuh8NFQ=";
```

Can we use `hash` here ?

---

Well, it builds and runs, but what is the difference between `hash` and `sha256` here? I couldn't find info on it both in manual and Discourse.

---

Indeed, there has been a significant push within the PR community for maintainers to transition from `sha256` to simply using `hash`. 

At present, there are no definitive rules or guidelines outlining how to write a derivation. Consequently, both notations are considered acceptable and functionally equivalent. If the proposed changes hadn't been implemented, I would have approved this PR in a matter of days anyway.

Now from a personal point of view, given the variety of potential hashing algorithms, it's not necessary to duplicate twice `sha256` in the line. This redundancy can be avoided by using `hash` instead with a SRI hash (https://www.w3.org/TR/SRI/).

Anyway, I'm glad you fixed it, I keep an eye on the PR and merge the PR once tests passed.